src/TurbulenceConvection/Parameters.jl

```diff
@@ -0,0 +1,68 @@
+"""
+    Parameters
+
+"""
+module Parameters
+
+import Thermodynamics as TD
+import SurfaceFluxes as SF
+import CloudMicrophysics as CM
+
+abstract type AbstractTurbulenceConvectionParameters end
+const ATCP = AbstractTurbulenceConvectionParameters
+
+#####
+##### TurbulenceConvection parameters
+#####
+
+Base.@kwdef struct TurbulenceConvectionParameters{FT, MP, SFP} <: ATCP
+    Omega::FT
+    planet_radius::FT
```

Comment:
Ditto for planet_radius: inherited from ClimaAtmos

Reply:
See this [comment](https://github.com/CliMA/ClimaAtmos.jl/pull/583#discussion_r915939028)